cypress/integration/sitewide-banner.js

```diff
@@ -0,0 +1,61 @@
+/// <reference types="Cypress" />
+
+import faker from 'faker';
+
+import { userFactory } from '../fixtures/user';
+import exampleCampaign from '../fixtures/contentful/exampleCampaign';
+
+describe('Site Wide Banner', () => {
+  beforeEach(() => {
+    cy.configureMocks();
+
+    cy.mockGraphqlOp('ContentfulBlockQuery', {
+      block: {
+        __typename: 'ContentBlock',
+        superTitle: faker.lorem.words(),
+        title: faker.company.bsBuzz(),
+        subTitle: faker.lorem.words(),
+        content: faker.lorem.sentence(),
+      },
+    });
+  });
+
+  it('The Site Wide Banner is displayed on campaign pages', () => {
+    cy.withState(exampleCampaign).visit('/us/campaigns/test-example-campaign');
```

Comment:
There's a fun helper for this!
```suggestion
    cy.anonVisitCampaign(exampleCampaign);
```

Reply:
yeeeeees! i shall use!